src/parser/mod.rs

```diff
@@ -8602,6 +8602,18 @@ impl<'a> Parser<'a> {
                 arg,
                 operator: FunctionArgOperator::Equals,
             })
+        } else if dialect_of!(self is DuckDbDialect) && self.peek_nth_token(1) == Token::Assignment
```

Comment:
Normally we also support Generic dialect as well for syntaxes. I took the liberty of adding this in another commit